entry/magento-os/package.json

```diff
@@ -11,12 +11,14 @@
         "webpack-cli": "^5.1.4"
     },
     "scripts": {
-        "build-js": "webpack",
+        "build-js": "webpack --config webpack.config.dev.js",
         "build-css": "npm run build-common-css && npm run build-lane-css",
         "build-common-css": "npx tailwindcss -i ../../generated/magento-os/compiled/building/tailwind/common.css -o ../../generated/magento-os/compiled/asset.building/build/style/common.css",
         "build-lane-css": "npx tailwindcss -i ../../generated/magento-os/compiled/building/tailwind/lane.css -o ../../generated/magento-os/compiled/asset.building/build/style/lane.css",
         "build": "webpack && npm run build-common-css -- --minify && npm run build-lane-css -- --minify && ./cli build.asset.sign",
-        "watch-js": "webpack --mode=development",
```

Comment:
regarding argument name, we need to align and use same argument for lightna build and assets build. I laready used --live argument which means that output will go to current build instead of temporary. 
--live might confise as can be assiciated with production.
--mode=development (lets say it will be just --dev) also might confuse as is associated with dev only which is forbidden on production by convention.

My another idea is to use --direct, then it won't be associated with cetraint environment